### ISSUE TITLE
Duplicate selectors are lost

### ISSUE THREAD

**Issue by MohammadYounes** [postcss/postcss-js#7]
```JS
var css = postcss.parse('div { color:blue;} div {padding:5px;}');
postcssJs.objectify(css); // --> { div: { padding: '5px' } }
```
> Related https://github.com/rtsao/postcss-rtlcss-combined/issues/9

**Comment by ai** [comment]
Fixed by @TrySound https://github.com/postcss/postcss-js/pull/8

**Comment by ai** [comment]
Released in 0.1.1
